deployment/terraform/live/vars.tf

```diff
@@ -162,6 +162,11 @@ variable "refinery_google_recaptcha_secret_key" {
   default     = ""
 }
 
+variable "refinery_intro" {
+  description = "Mark-down to be displayed in the intro paragraph on the landing page."
+  default = "The refinery platform is for managing, analyzing, and visualizing data belonging to members of the [Harvard Stem Cell Institute (HSCI)](https://hsci.harvard.edu/).\s\s Users can upload various types of files, annotated with pertinent metadata, and can choose to share those files within their own lab, with collaborators. Users can also run common analyses via customizable [Galaxy](https://galaxyproject.org/) workflows and explore subsequent results with interactive visualization tools.\s\s Watch feature-specific tutorial videos below to get started. Once you have registered an account, log in, and click [My Dashboard](/dashboard) to start managing your own workspace."
```

Comment:
this hsci reference was a typo, this is the generic paragraph.

Reply:
It still says `belonging to members of the [Harvard Stem Cell Institute (HSCI)](https://hsci.harvard.edu/)`?

Reply:
@hackdna Yes, that was my point. But regardless this is a placeholder paragraph until Nils and Shannan have time to write something up.